packages/opentelemetry-context-base/src/types.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Default type for functions */
+// tslint:disable:no-any
+export type Func<T> = (...args: any[]) => T;
+
+/** A key/value object for anything that can be stored in the context */
+export interface ContextStore {
+  [key: string]: unknown;
```

Comment:
I believe the simpler is the API, the better it is. So instead of having a custom class and `get`/`set` method i choose to just expose a object there.